### PR TITLE
Release of Greykite 1.0.0

### PR DESCRIPTION
**Pull request by sayanpatra**
Adds Greykite Anomaly Detection module along with other feature updates and bugfixes.

### CODE REVIEW

**Review by Reza1317 — APPROVED** [review]
Thanks. LGTM.

(Already had reviewed internally.)

File: HISTORY.rst [hunk 1]
```diff
@@ -2,10 +2,41 @@
...
+For example user can target a minimal recall level of 80% while maximizing precision. Additionally, the users can specify a 
+minimum error level to filter out anomalies that are not business relevant. The motivation to include criteria other than 
+statistical significance is to bake in material/ business impact into the detection.
+
+    * @Reza Hosseini: Devised the core anomaly detection library structure. Added base ``Detector`` module. 
+    * @Reza Hosseini: Added `~greykite.detection.detector.reward.Reward` that allows users to specify and optimize robust anomaly detection objectives.
+    * @Sayan Patra: Added ``GreykiteDetector`` module that builds anomaly detection based on Greykite forecasting.
+    * @Sayan Patra: Added tutorials for Greykite anomaly detection.
+
+* New features and methodss
```
Comment (njusu) [inline HISTORY.rst:21]: typo: methods

File: HISTORY.rst [hunk 2]
```diff
@@ -2,10 +2,41 @@
...
+    * @Reza Hosseini: Added `~greykite.common.features.outlier.ZScoreOutlierDetector` and `~greykite.common.features.outlier.TukeyOutlierDetector`, improved outlier detection modules. 
+    * @Sayan Patra: Added `~greykite.detection.common.pickler.GreykitePickler`. This improves the pickling function for Greykite models and allows to store the model in a single file.
+    * @Yi-Wei Lu: Added ``DifferenceBasedOutlierTransformer`` that can identify outliers in the ``sklearn`` pipeline.
+
+* Library Enhancements
```
Comment (njusu) [inline HISTORY.rst:26]: Enhancements -> lower case

File: HISTORY.rst [hunk 3]
```diff
@@ -2,10 +2,41 @@
...
+    * @Kaixu Yang: Added ``scipy`` solver to make quantile regression more stable.
+    * @Qiang Fei: Updated ``auto_holiday`` functionality to use holiday groupers for improved forecast performance in holiday periods.
+    * @Katherine Li: Improved changepoint detection method that can identify level shift.
```
Comment (njusu) [inline HISTORY.rst:29]: shift -> shifts

**Review by njusu — COMMENTED** [review]
Reviewed the general files only as the code scripts were reviewed internally. Overall looks good, just a few minor comments.

**Review by njusu — APPROVED** [review]
LGTM

**Pull request merged**